code-challenges.test.js

```diff
@@ -14,56 +16,155 @@
+
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("takes in an array of numbers and returns an array with all the numbers multipled by 3.", () => {
+
+        const numbersArray1 = [6, 7, 8, 9, 10]
+        // Expected output: [18, 21, 24, 27, 30]
+        const numbersArray2 = [24, 27, 30, 33, 36]
+        // Expected output: [72, 81, 90, 99, 108]
+  
+      //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(multThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+      expect(multThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+    })
+  })
+// Got two red 
 
 
 // b) Create the function that makes the test pass.
 
+// create a function named multThree 
+// map through the array 
+// multiply by 3 for each value 
+// return 
+
 
+const multThree = (array) => {
+    return array.map(value => value * 3)
+}
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
-// Expected output: "15 is divisible by three"
-const num2 = 0
-// Expected output: "0 is divisible by three"
-const num3 = -7
-// Expected output: "-7 is not divisible by three"
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("evenThree", () => {
+
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("takes a number as an argument and decides if the number is evenly divisble by three or not.", () => {
+
+        const num1 = 15
+        // Expected output: "15 is divisible by three"
+        const num2 = 0
+        // Expected output: "0 is divisible by three"
+        const num3 = -7
+        // Expected output: "-7 is not divisible by three"
+  
+      //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(evenThree(num1)).toEqual("15 is divisible by three")
+      expect(evenThree(num2)).toEqual("0 is divisible by three")
+      expect(evenThree(num3)).toEqual("-7 is not divisible by three")
+    })
+  })
+
+// Got two red 
 
 
 // b) Create the function that makes the test pass.
 
+// create a function evenThree with number as an argument
+// use conditionals to determine if number is divisible by 3
+// return the number is divisible by three using interpolation
+// return numbers that are not disvisible by 3
+
+const evenThree = (number) => {
+    if (number % 3 === 0){
+        return `${number} is divisible by three`
+    } else {
+        (number  % 3 === 0)
+        return `${number} is not divisible by three`
+    }
+    
+}
```

Comment:
Logic looks great and excellent pseudocode!  If you want to try a different approach, I would recommend researching ternary operators.